pkg/report/writer.go

```diff
@@ -115,10 +118,18 @@ func Write(report Report, option Option) error {
 	case "json":
 		writer = &JSONWriter{Output: option.Output}
 	case "template":
+		// We should keep `sarif.tpl` template for backward compatibility for a while.
```

Comment:
We don't keep the template as we remove, but we keep it working as is.

Reply:
changed this comment.